python/utils/gpu_metric_poller.py

```diff
@@ -31,7 +31,7 @@
 import os
 import sys
 import threading
-from pynvml import smi
+from pynvml_utils import smi
```

Comment:
Pretty sure we can just do `import pynvml`, and replace all `smi.` instances below with `pynvml.`.

In other words, non of the logic in this file is actually using "smi". As far as I can tell, the only reason this works now is that `smi.py` includes a `from pynvml import *` statement.

Reply:
Not sure this is true. Am seeing quite a lot of `smi` usage in this file. For example

https://github.com/rapidsai/cugraph/blob/e3054167d596927f94c5a62fba773d4d144b796a/python/utils/gpu_metric_poller.py#L94-L98

Reply:
Right - I'm saying that all those calls are actually using `pynvml` (not `smi`).

For example, calling `smi.nvmlInit()` is actually calling `pynvml.nvmlInint()` (the main hint is that all commands start with `nvml`).

Reply:
Ah ok. Thanks for clarifying! 🙏

Have made this change below